test/proposals.test.js

```diff
@@ -47,6 +49,9 @@ describe('Proposals', async assert => {
   await contracts.accounts.adduser(thirduser, 'thirduser', 'individual', { authorization: `${accounts}@active` })
   await contracts.accounts.adduser(fourthuser, 'fourthuser', 'individual', { authorization: `${accounts}@active` })
 
+  await contracts.accounts.testresident(firstuser, { authorization: `${accounts}@active` })
+  await contracts.accounts.testresident(seconduser, { authorization: `${accounts}@active` })
```

Comment:
I think the `fourthuser` should be included as a resident too, because below this user creates a proposal below, and this test is failing due to that.